### PR TITLE
Chat grammar

### PR DESCRIPTION
**Pull request by ArtOfCode-**
Better grammatical correctness in some of the lesser-seen messages.

### CODE REVIEW

File: chatcommunicate.py [hunk 1]
```diff
@@ -330,18 +330,18 @@ def handle_commands(content_lower, message_parts, ev_room, ev_room_name, ev_user
...
         if len(message_parts) < 2:
             return False, "Not enough arguments."
         output = []
         index = 0
         urls = list(set(message_parts[1:]))
         if len(urls) > 5:
-            return False, "To avoid that SmokeDetector reports posts too slowly, " \
-                          "you can report maximally 5 posts at a time. " \
-                          "This is to avoid that SmokeDetector's chat messages get rate-limited too much, " \
+            return False, "To avoid that SmokeDetector reporting posts too slowly, " \
```
Comment (Manishearth) [inline chatcommunicate.py]: no "that"

Comment (ArtOfCode-) [inline chatcommunicate.py]: @Manishearth damnit, thought I caught all of those. Gimme a minute.

**Comment by Manishearth** [comment]
Tests need an update https://circleci.com/gh/Charcoal-SE/SmokeDetector/1777

**Comment by ArtOfCode-** [comment]
@Manishearth was already on that ;) all pass now

**Pull request merged**